app/models/document.rb

```diff
@@ -185,6 +172,10 @@ def facet_options(facet)
 
   delegate :organisations, to: :finder_schema
 
+  def primary_publishing_organisation
+    organisations.first
```

Comment:
Is this a safe assumption? Just double checking

Reply:
Hmmm I'll take another look at that :/

Reply:
Just deployed the branch to integration and was able to change the primary publishing org on https://specialist-publisher.integration.publishing.service.gov.uk/flood-and-coastal-erosion-risk-management-research-reports/6993f8c2-8936-492c-a5c5-4ea4b7444f97:en - so I think we're ok.